Simplenote/AuthViewController.h

```diff
@@ -52,6 +52,5 @@
 - (void)presentPasswordResetAlert;
 - (void)presentPasswordCompromisedAlert;
 - (void)presentUnverifiedEmailAlert;
-- (void)showAuthenticationErrorForCode:(NSInteger)responseCode responseString:(NSString *)responseString;
```

Comment:
This was triggering a warning. I've missed dropping the forward declaration, when ported to Swift